src/vl-map-wms-layer.js

```diff
@@ -52,11 +51,10 @@ export class VlMapWmsLayer extends VlMapLayer {
   }
 
   get _sldBody() {
-    return this.getAttribute('data-vl-sld-body') ? this.__sldBodyWithWhitespaceRemoved : '';
-  }
-
-  get __sldBodyWithWhitespaceRemoved() {
-    return this.getAttribute('data-vl-sld-body').replace(/>\s*/g, '>').replace(/\s*</g, '<');
+    const wmsStyle = this.querySelector(':scope > vl-map-wms-style');
+    if (wmsStyle && wmsStyle.getAttribute('data-vl-sld') != null) {
```

Comment:
zou die getAttribute niet best weggeabstraheerd worden in vl-map-wms-style?

Reply:
Leek niet direct te werken. Ik wou het eerst via parentElement zetten, maar probleem is dat die stijl al direct nodig is om de wms layers aan te maken. En bij connectedCallback was de functie op vl-map-wms-style nog niet beschikbaar. Enig idee hoe ik dit zou kunnen oplossen dan?

Reply:
Eventueel `vl-map-wms-layer` de `vl-map-wms-style` laten importeren en wachten op de definition van `vl-map-wms-style` in de custom elements registry vooraleer `vl-map-wms-layer` defined/verwerkt wordt. Dan zou de functie op vl-map-wms-style beschikbaar moeten zijn.

Reply:
Merci, lijkt te werken!